internal/service/ocm/ocm_impl_test.go

```diff
@@ -0,0 +1,59 @@
+package ocm
+
+import (
+	"os"
+	"testing"
+
+	"github.com/redhatinsights/mbop/internal/config"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestSuite struct {
```

Comment:
Also learning as we go - apparently this should be named something like "TestIsInternalSuite" - it makes the logs better since they actually print out what they're testing if you pass the `-v` flag.